src/io/parsers/VolumeModel.js

```diff
@@ -95,8 +97,9 @@ class VolumeModel {
     const z2 = (Math.cos(header.angles.x) - Math.cos(header.angles.y)
       * Math.cos(header.angles.z)) / Math.sin(header.angles.z);
     const z3 = Math.sqrt(1.0 - z1 * z1 - z2 * z2);
+
     const xaxis = new THREE.Vector3(xScale, 0, 0);
-    const yaxis = new THREE.Vector3(Math.cos(header.angles.y) * yScale, Math.sin(header.angles.y) * yScale, 0);
+    const yaxis = new THREE.Vector3(Math.cos(header.angles.z) * yScale, Math.sin(header.angles.z) * yScale, 0);
```

Comment:
Is it a bugfix? It introduces a new question then. Why do we call it `yaxis` if it is calculated from `z` angle? Is it possible to adjust variable names to be consistent?

Reply:
Yes, there was the bug. Adjusted variable names in a new commit.